client/lib/menu-data/menu-data.js

```diff
@@ -438,8 +447,8 @@ MenuData.prototype.deleteMenu = function( menu, callback ) {
 };
 
 /**
- * @param {string} location to restore the menu to
- * @param {function} [optional] callback
+ * @param {String} location to restore the menu to
```

Comment:
I like separating the parameter name from the description with a hyphen, like you're doing in other places.